js/foundation.smoothScroll.js

```diff
@@ -48,10 +48,8 @@ class SmoothScroll extends Plugin {
 
         // click handler function.
         var handleLinkClick = function(e) {
-            // exit function if the event source isn't coming from an anchor with href attribute starts with '#'
-            if(!$(this).is('a[href^="#"]'))  {
-                return false;
-            }
+            // Follow the link it does not point to an anchor.
```

Comment:
it => if it

Probably better: if it is not an anchor

Reply:
> Probably better: if it is not an anchor

There is an hyperlink, and what the link points to, an anchor. We are not testing if the hyperlink is an anchor (with `href="#xxx"`) but it points to it.

MDN and W3C calls these elements "Anchor Element" and "fragment id", but everywhere else the "boorkmark" with `href="#xxx"` is called an anchor, and I never heard of "fragment id" before today 😄